src/assets/scripts/service-k3s.initd

```diff
@@ -33,6 +33,15 @@ depend() {
 start_pre() {
   rm -f /tmp/k3s.*
   checkpath --file --mode 0644 --owner root "${output_log_unquoted}" "${error_log_unquoted}"
+  ARCH=amd64
+  if [ "$(uname -m)" = "aarch64" ]; then
+    ARCH=arm64
+  fi
+  if [ "${ENGINE}" == "moby" ]; then
+    docker load </var/lib/rancher/k3s/agent/images/k3s-airgap-images-${ARCH}.tar
+  else
+    nerdctl -n k8s.io load </var/lib/rancher/k3s/agent/images/k3s-airgap-images-${ARCH}.tar
```

Comment:
We'll need to add `need containerd` to `depend` in this case.

Should probably wait for #1588 to land first.

Separately, consider using `--input` instead of reading from stdin (this _shouldn't_ be an issue, but I'm always afraid of random things converting `\n` to `\r\n`…)

Reply:
>Should probably wait for https://github.com/rancher-sandbox/rancher-desktop/pull/1588 to land first.

Yes, seemed like a good idea. I tried to rebase on top of it and found out that #1588 doesn't work yet; it is missing the `loopback` and `flannel` cni plugins (see #1639).

>Separately, consider using --input instead of reading from stdin (this shouldn't be an issue, but I'm always afraid of random things converting \n to \r\n…)

I have done this now. Also had to add `-H /run/k3s/containerd/containerd.sock` to `nerdctl` so it can find the socket.